tests/unit/bokeh/embed/test_bundle.py

```diff
@@ -131,7 +131,8 @@ def setup_class(cls):
 
     @classmethod
     def teardown_class(cls):
-        del Model.model_class_reverse_map["latex_label.LatexLabel"]
+        from latex_label import LatexLabel
+        _default_resolver.remove(LatexLabel)
```

Comment:
I was just trying with `Model._clear_extensions()` here and it did not work, shoudn't it though?

Reply:
I am going to build the docs again and check the custom extension examples autoload bundles to make sure they are not accumulating previous extension definitions.

Reply:
Everything looks correct in the autoload bundles so I'll merge this after CI is done and start the release build. I would like to understand why `Model._clear_extensions()` did not fix the tests, though.  I'll also open a follow-up issue. As long as we are factoring this out to a separate component we should add tests an also improve it so that hacks like `_clear_extensions` are not necessary.

Reply:
There are two kinds of extensions and `Model._clear_extensions()` only clears one kind. All of this needs a proper redesign and a clean up at some point.